app/assets/javascripts/app/core/mobile_detect.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  'use strict';
+
+  angular
+  .module('council.core')
+  .factory('mobileDetect', mobileDetect);
```

Comment:
It's a service, not a factory, factory is for creating instances. And the factory is called `mobileDetect` and the method is `isMobile`, so the a common usage would be `mobileDetect.isMobile()`, witch is acceptable, not good but acceptable. But what about if you wan't to ask if it's tablet or desktop? `mobileDetect.isDesktop()` doesn't look like a good name.